### DISCUSSION TITLE
Why not little more easy UI

### DISCUSSION THREAD

**Discussion by tmprabubiz** [lllyasviel/Fooocus#3148] [Q&A] [answered]
I am an avid fan of Illayasviel's Fooocus. I have 3 doubts

Doubt-1
In Presets, if an option like "CUSTOM" is provided with the provision to change the Sampler, Scheduler, CFG and Generation Steps in the first page itself.  Or a pop down menu of the same to use with any presets will also make things easy. Going to 'Developer Debug Mode'  for every model provides chances to make mistakes for non-technical users like me.

Doubt-2
Similar to the excellent preset Playground_v2.5, may you provide the SD3 preset too.

Doubt - 3
In all Stable Diffusion applications, either the user can create a single object or a crowd, of course with visual attributes. By any chance, in the future, will you increase the possibilities of generating the different scenic compositions with 2 to 4 subjects in action with a user specified background.  That will provide the opportunity to generate different visuals instead of single human, single animal, single bird, futuristic city, alien, robot, statues or sculptures and landscapes. The basic craft of the image creation is not symmetrical and uniformly balanced images. Instead, the abstractness should be emphasized for what the art form is meant for. I feel every one in AI - Text to Image creation are mostly doing different types of portraits.

Correct me, if I am wrong!

Regards

**Comment by newxhy** [comment]
我觉得你需要的是StableSwarmUI

**Reply by tmprabubiz** [reply]
Hi newxhy
Kindly do not imagine within your restricted knowledge. I have asked a doubt within this Fooocus discussion group. Let the respective developers and experts know the concerns of the users and make necessary arrangements Fooocus itself.

_______
请不要在你有限的知识范围内想象。我在这个讨论组里提出了一个疑问。让各自的开发者和专家了解用户所关心的问题并做出必要的安排。（抱歉，如果我的中文错误，我已经使用谷歌翻译来回答你）

**Comment by mashb1t** [comment, marked as answer]
Hey @tmprabubiz,

1.Feel free to adjust the presets to your liking. Presets always override all values in them. A preset creation from currently selected settings is in the backlog.
2. it (sadly) needs a lot more than just adding a new preset. See https://github.com/lllyasviel/Fooocus/discussions/3145#discussioncomment-9773154
3. If your goal is to replace the background check out the inpaint mask feature). In my fork i have also already included an [auto mask feature](https://github.com/lllyasviel/Fooocus/pull/2032) + am currently working on https://github.com/lllyasviel/Fooocus/discussions/3122. Did i get your intention on this item correctly?

**Reply by tmprabubiz** [reply]
Thank You for the response
1. I am not a technical person to do coding, rather I can alter the preset if I am guided step by step.
2. The details in the link pages provided in your answer are too technical for me
However, I appreciate the response
